lib/chef/knife/bootstrap_windows_winrm.rb

```diff
@@ -37,6 +41,17 @@ class BootstrapWindowsWinrm < Bootstrap
       banner "knife bootstrap windows winrm FQDN (options)"
 
       def run
+        if (Chef::Config[:validation_key] && !File.exist?(File.expand_path(Chef::Config[:validation_key])))
+
+          if !negotiate_auth? && !(locate_config_value(:winrm_transport) == 'ssl')
+            ui.error("Validatorless bootstrap only supported with negotiate authentication protocol and ssl/plaintext transport")
```

Comment:
My guess is that without SSL or negotiate being used, the payload would be sent unencrypted over the wire - which would include the user.pem.  

However, this is a false assumption, since Negotiate should work with plaintext transport without encryption as well (as it does with  the winrs command).  

That assumption (which winrm-s currently makes) is why we get an error with negotiate when the remote node has allowunencrypted enabled.